Convert.h

```diff
@@ -113,35 +107,33 @@ class Convert {
                         error = -1;
                     }
                 } else {
-                    total += map[romanNum1];
+                    total += RomanToArabic[romanNum1];
                     romanNum1 = romanNum2;
                 }
             }
             finish(total, error);
             total = 0;
             error = 0;
         }
-        total += map[romanNum1];
+        total += RomanToArabic[romanNum1];
         finish(total, error);
     }
 
     void finish(int total, int error) {
-        ofstream SetFile(oFileNameS);
+        ofstream SetFile(oFileName);
         if (error == 0) {
             SetFile << originalRomanNums.front() << " || " << total << endl;
         } else {
             SetFile << originalRomanNums.front() << " || " << total << "There was an error calculating this value" << endl;
         }
     }
 
-    char *iFileName;
-    char *oFileName;
-    string iFileNameS;
-    string oFileNameS;
+    string iFileName;
+    string oFileName;
     queue<char> romanNumerals;
     queue<int> convertedValues;
     queue<string> originalRomanNums;
-    map<string, int> map;
+    map<string, int> RomanToArabic;
```

Comment:
my compiler threw an error while trying to compile this. I don't think type names and variables can be the same in modern C++ versions. What version were you using?